docs/windows/deployment/publish-unpackaged-cli.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Use the CLI to publish Unpackaged apps for Windows"
+description: "Learn how to package and publish an unpackaged Windows .NET MAUI app with the dotnet publish command."
+ms.date: 10/12/2022
+---
+
+# Publish an unpackaged .NET MAUI app for Windows with the CLI
+
+> [!div class="op_single_selector"]
+>
+> - [Publish Packaged / MSIX](publish-cli.md)
+> - [Publish Unpackaged / EXE](publish-unpackaged-cli.md)
+
+When distributing your .NET Multi-platform App UI (.NET MAUI) app for Windows, you can publish the app and its dependencies to a folder for deployment to another system.
+
+## Configure the project build settings
+
+The project file is a good place to put Windows-specific build settings. You may not want to put some settings into the project file, such as passwords. The settings described in this section can be passed on the command line with the `-p:name=value` format. If the setting is already defined in the project file, a setting passed on the command line overrides the project setting.
```

Comment:
Should we remove this paragraph for unpackaged, to simplify the steps? For unpackaged, there are no passwords. And I don't see any instructions below that give other settings that should go in the project file (except for RuntimeIdentifier, which I assume has to go there).